pkg/common/namespace_selection.go

```diff
@@ -1,48 +0,0 @@
-// Copyright (c) 2020 Red Hat, Inc.
-// Copyright Contributors to the Open Cluster Management project
-
-package common
-
-import (
-	"context"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-)
-
-// GetSelectedNamespaces returns the list of filtered namespaces according to the policy namespace selector
-func GetSelectedNamespaces(included, excluded, allNamespaces []string) []string {
```

Comment:
One other thing I noticed while poking at this code. I deleted `namespace_selection.go` since it wasn't being used, but maybe I should have migrated the functions there instead and deleted the functions from the controller? It'd be a straightforward change (and would make the controller file a bit shorter). Thoughts?

Reply:
+1 to smaller files.